src/puppetlabs/puppetdb/cli/services.clj

```diff
@@ -355,6 +356,23 @@
           (get-in (service-context this) [:shared-globals :scf-read-db])
           row-callback-fn)))
 
+(defprotocol MetricsServer)
```

Comment:
Can we break this stuff into its own file somewhere? This would aid with ripping it out later on when we choose to.